packages/client/lib/execution/receipt.ts

```diff
@@ -159,7 +159,7 @@ export class ReceiptsManager extends MetaDBManager {
   async getLogs(
     from: Block,
     to: Block,
-    address?: Buffer,
+    addresses?: Buffer[],
```

Comment:
Not daring a review atm, not "into" client dev and context sufficiently enough atm. But first read question: isn't this change now preventing the original use case just passing in a single address? 🤔

Reply:
Yes, obviously, but the question was rather if it should nevertheless still allow passing in as a non-array address?

The [spec](https://eth.wiki/json-rpc/API#eth_getlogs) says:

> address: DATA|Array, 20 Bytes - (optional) Contract address or a list of addresses from which logs should originate

So if I read correctly I guess this allows both and typing should therefore be rather: `Buffer|Buffer[]` ? 🤔

Reply:
Ah ok, thanks!